server/pbench/bin/pbench-move-unpacked

```diff
@@ -44,7 +44,7 @@ linkdest=MOVED-UNPACKED
 echo $TS
 
 # get the list of files we'll be operating on - sort them by size
-list=$UNPACK_PATH/pbench-unpack-tarballs.$$
+list=$UNPACK_PATH/$PROG.$$
```

Comment:
Isn't there a problem here? $UNPACK_PATH and $TMP might be the same thing, in which case there will be a conflict.

Reply:
No I have tested it on 144 and is working fine because list variable will create a file, and unpack path is a directory under tmp. Let me know if you think differently.